client/my-sites/google-my-business/stats/index.js

```diff
@@ -0,0 +1,124 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { random, range } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import StatsFirstView from 'my-sites/stats/stats-first-view';
+import StatsNavigation from 'blocks/stats-navigation';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+import { getSelectedSiteSlug, getSelectedSiteId } from 'state/ui/selectors';
+import DocumentHead from 'components/data/document-head';
+import Card from 'components/card';
+import CardHeading from 'components/card-heading';
+import SectionHeader from 'components/section-header';
+import PieChart from 'components/pie-chart';
+import Chart from 'components/chart';
+import SearchDataType from './search-data-type';
+
+class GoogleMyBusinessStats extends Component {
+	static propTypes = {
+		translate: PropTypes.func.isRequired,
+		siteSlug: PropTypes.string.isRequired,
+		siteId: PropTypes.number.isRequired,
+		searchData: PropTypes.arrayOf( SearchDataType ).isRequired,
+		searchDataTotal: PropTypes.number.isRequired,
+		viewData: PropTypes.array.isRequired,
+		actionData: PropTypes.array.isRequired,
+	};
+
+	render() {
+		const {
+			translate,
+			siteSlug,
+			siteId,
+			searchData,
+			searchDataTotal,
+			viewData,
+			actionData,
+		} = this.props;
+
+		return (
+			<Main wideLayout>
+				<DocumentHead title={ translate( 'Stats' ) } />
+				<StatsFirstView />
```

Comment:
I'm not entirely sure we need that component since it doesn't really make sense within Google My Business:

> The Stats page tells you everything you need to know about your site's traffic and visitors — which of your posts are taking off, where your audience is coming from, and who's now following you.

Reply:
Yeah, I pretty blindly copied the `Insights` tab. This component definitely can go.